### PR TITLE
Help docs

### PR DESCRIPTION
**Pull request by deetz99**
Initial cut at help docs

### CODE REVIEW

File: web/site/app.vue [hunk 1]
```diff
@@ -14,6 +15,13 @@ useHead({
 })
 
 const appVersion = await getAppMetaInfo()
+
+const { data: helpDocs } = await useAsyncData('help-docs-fetch', () => {
```
Comment (deetz99) [inline web/site/app.vue:19]: query help docs in app.vue and provide globally. Originally I had this in the SbcHelpContent component but the prerender didnt work inside the modal

File: web/site/components/Sbc/Help/Content.vue
Comment (deetz99) [inline web/site/components/Sbc/Help/Content.vue:1]: get help docs from app.vue provide and display in ContentRenderer component

File: web/site/components/Sbc/Help/Modal/index.vue
Comment (deetz99) [inline web/site/components/Sbc/Help/Modal/index.vue:1]: modal for displaying help docs

File: web/site/components/Sbc/Help/Trigger.vue
Comment (deetz99) [inline web/site/components/Sbc/Help/Trigger.vue:1]: help docs button, renders a link on desktop or a button to open modal for touchscreen

File: web/site/components/content/ProseA.vue [hunk 2]
```diff
@@ -33,7 +33,10 @@ function resolvePath () {
     :target
     :download
     variant="link"
-    class="-m-6"
+    class="-mx-2 -my-4"
+    size="sm"
+    trailing
+    :icon="target === '_blank' ? 'i-mdi-open-in-new' : ''"
```
Comment (deetz99) [inline web/site/components/content/ProseA.vue:39]: add icon to proseA component if target === _blank

File: web/site/composables/useHelpModal.ts
Comment (deetz99) [inline web/site/composables/useHelpModal.ts:1]: handle help docs modal state, will come in use if opening docs to certain section

File: web/site/components/Sbc/Help/Content.vue
Comment (deetz99) [inline web/site/components/Sbc/Help/Content.vue:1]: display help docs from injected value

File: web/site/components/Sbc/Help/Modal/index.vue
Comment (deetz99) [inline web/site/components/Sbc/Help/Modal/index.vue:1]: modal to display help text

File: web/site/components/Sbc/Help/Trigger.vue
Comment (deetz99) [inline web/site/components/Sbc/Help/Trigger.vue:1]: button to trigger help text, will render anchor for desktop or a button to open modal for touchscreen

**Review by thorwolpert — APPROVED** [review]

File: web/site/components/content/ProseH1.vue
Comment (deetz99) [inline web/site/components/content/ProseH1.vue:1]: remove prose heading overwrites

File: web/site/composables/useHelpModal.ts
Comment (deetz99) [inline web/site/composables/useHelpModal.ts:1]: composable to track help modal state

File: web/site/nuxt.config.ts
Comment (deetz99) [inline web/site/nuxt.config.ts:1]: set anchorLinks config for nuxt content

**Pull request merged**